### PR TITLE
CORCI-768 build: Reset @Library branch for testing

### PR DESCRIPTION
**Pull request by JohnMalmberg**
Reset the @Library branch for testing back to default.

Signed-off-by: John E Malmberg <john.e.malmberg@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -25,7 +25,7 @@
 // Then a second PR submitted to comment out the @Library line, and when it
 // is landed, both PR branches can be deleted.
 
-@Library(value="pipeline-lib@corci-768a") _
+//@Library(value="pipeline-lib@corci-768a") _
```
Comment (brianjmurrell) [inline Jenkinsfile]:
```suggestion
//@Library(value="pipeline-lib@your_branch") _
```

To get back to it's original value and a value that is consistent with every other `Jenkinsfile`.

**Comment by JohnMalmberg** [comment]
So using the commit feature from github fails the DCO check.

**Comment by brianjmurrell** [comment]
> So using the commit feature from github fails the DCO check.

If you mean editing and committing changes from within the GitHub UI, you need to add the SoB: yourself to your commit message.

**Comment by JohnMalmberg** [comment]
Superseded by https://github.com/daos-stack/pipeline-lib/pull/71

**Pull request closed without merging**